### ISSUE TITLE
New currencies alert doesn't show the currency name (if available)

### ISSUE THREAD

**Issue by AlexITC** [AlexITC/crypto-coin-alerts#12]
New currencies alert doesn't show the currency name (if available)

### Expected behavior

While receiving the new currencies, the currency name is displayed (if available).

### Actual behavior

The currency name is not being displayed.

### Steps to reproduce the behavior

Receive a new currencies alert from COINMARKETCAP, even that the currency name is available, it is not displayed.

**Comment by mghildiy** [comment]
Hi,

How to use COINMARKETCAP?

**Comment by AlexITC** [comment]
Adding more details on how to verify the behavior:
- Login
- Go to the new fixed price alert page: https://cryptocoinalerts.net/new-fixed-price-alert
- Set Exchange to `COINMARKETCAP`
- Set Market to `USD`.
- Set Currency to `BTC`
- Set Price to `1000`
- Set Price condition to `Above`
- Click on Create alert.
- Wait for the alert to be triggered (around 5 minutes) and you'll have the alert in your email.

**Comment by mghildiy** [comment]
I got this in alert:

COINMARKETCAP:
BTC (Bitcoin) has increased to 6475.36566404 USD
 
I see BTC, USD mentioned.
What is missing in this alert message?

**Comment by AlexITC** [comment]
That's the supposed alert that we need to receive, if you look carefully, the `(Bitcoin)` is the currency name.

**Comment by AlexITC** [comment]
@mghildiy The process that I described is wrong.

- Go to https://cryptocoinalerts.net/new-currency-alerts
- Be sure `COINMARKETCAP` is checked

When there is a new alert, the coin symbol is displayed but the name is not.

**Comment by mghildiy** [comment]
This is the latest alert I received:

Trade DIVX on BTC market.
Trade DIVX on USD market.

**Comment by AlexITC** [comment]
You should have received:

```
Trade DIVX (Divi Exchange Token) on BTC market.
Trade DIVX (Divi Exchange Token) on USD market.
```

**Comment by AlexITC** [comment]
Fixed by #33
